src/test/java/com/dubture/jenkins/digitalocean/DigitalOceanCloudTest.java

```diff
@@ -24,32 +24,36 @@
 
 package com.dubture.jenkins.digitalocean;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 
-public class DigitalOceanCloudTest {
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-    /*
-    @Rule
-    public JenkinsRule jenkinsRule = new JenkinsRule();
-    */
+@WithJenkins
+class DigitalOceanCloudTest {
```

Comment:
i don't see any reason to keep this test? it doesn't do anything at all does it?